internal/server/cmd_meta.go

```diff
@@ -329,6 +329,27 @@ var (
 		CmdType: SingleShard,
 	}
 
+	jsonforgetCmdMeta = CmdsMeta{
+		Cmd:     "JSON.FORGET",
+		CmdType: SingleShard,
+	}
+	jsondelCmdMeta = CmdsMeta{
+		Cmd:     "JSON.DEL",
+		CmdType: SingleShard,
+	}
+	jsontoggleCmdMeta = CmdsMeta{
```

Comment:
Lets keep the naming convention consistent here with other json commands. for ex `jsonForgetCmdMeta` .